examples/plugins/permission-boundary.js

```diff
@@ -20,7 +20,7 @@ class PermissionBoundaryAspect {
   }
 
   visit(node) {
-    if (node instanceof iam_role.IamRole) {
+    if (node.terraformResourceType === "aws_iam_role") {
       node.permissionsBoundary = this.permissionBoundaryArn;
     }
```

Comment:
The constructs created by the SDK are no longer the same classes as those provided by `@cdktf/provider-aws` so `instanceof` will fail - hence this change